src/collections/_documentation/platforms/android/index.md

```diff
@@ -0,0 +1,597 @@
+
+[{% asset android/android_error.png alt="Example of a caught exception in the Sentry dashboard" %}]({% asset android/android_error.png @path %})
+
+If you prefer learning with examples, you can download our [sample application](https://github.com/getsentry/sentry-android/tree/master/sentry-sample).
+
+## Manual Initialization
+
+Initialize the SDK manually when you need to provide additional configuration to the SDK that cannot be specified in the manifest.
+
+To initialize the SDK manually, disable the auto initialization. You can do so by adding the following line into your manifest:
+
+```xml
+<meta-data android:name="io.sentry.auto-init" android:value="false" />
+```
+
+The next step is to initialize the SDK directly in your code.
+
+The SDK can catch errors and crashes only after you've initialized it. So, we recommend calling `SentryAndroid.init` in the instance of the Application class right after the application is created. 
+
+Configuration options will be loaded from the manifest so that you don't need to have the static properties in your code. In the `init` method, you can provide a callback that will modify the configuration and also register new options.
+
+```java
+import android.app.Application;
+import io.sentry.android.core.SentryAndroid;
+import io.sentry.core.SentryLevel;
+
+public class SentryApplication extends Application {
+    public void onCreate() {
+        super.onCreate();
+        /**
+            * Manual Initialisation of the Sentry Android SDK
+            * @Context - Instance of the Android Context
+            * @Options - Call back function that you need to provide to be able to modify the options.
+            * The call back function is provided with the options loaded from the manifest.
+            *
+            */
+        SentryAndroid.init(this, options -> {
+            // Add a callback that will be used before the event is sent to Sentry.
+            // With this callback, you can modify the event or, when returning null, also discard the event.
+            options.setBeforeSend((event, hint) -> {
+                if (event.getLevel().equals(SentryLevel.DEBUG))
+                    return null;
+                else
+                    return event;
+            });
+        });
+    }
+}
+```
+
+For more details about the available configurations, see [Configuration options](#configuration-options).
+
+## Capturing Errors
+
+### Crash Reporting
+
+Your application will crash whenever a thrown exception goes uncaught. The Sentry SDK catches the exception right before the crash and builds a crash report that will persist to the disk. The SDK will try to send the report right after the crash, but since the environment may be unstable at the crash time, the report is guaranteed to send once the application is started again.
+
+If there is a fatal error in your native code, the process is the same. The difference is, a restart is always needed, as a hard crash immediately stops the app.
```

Comment:
This is not accurate. On my tests we consistently sent the event before the app crashed.
so for native crashes we might send before restart

Reply:
@bruno-garcia How about:

If there is a fatal error in your native code, the process is similar. The crash report will send before the app crashes.

Reply:
I'd just use might instead of will.
we will for sure send on restart but we'll attempt to sent it before the app goes away

Reply:
for fatal errors on native-code, I've always needed to restart it, it was never time enough to send it over.